architecture/adr-list/adr_002_identity_transactions.md

```diff
@@ -44,103 +44,191 @@ _**Note**: Hyperledger Indy also contains other transaction types beyond the one
 
-`id -> {(alias, id, verkey), last_tx_hash, last_update_timestamp }`
+1. **`id` (string):**
 
-_DID transaction format:_
+   A string with format `<DIDDoc-id>#<key-alias`
 
-```text
-Did
-{
-    "alias": "Alice DID",
-    "dest": "GEzcdDLhCpGCYRHW82kjHd",
-    "verkey": "~HmUWn928bnFT6Ephf65YXv"
-}
+2. **`controller` (list of base58-encoded string):**
+
+   A set of identifier strings.
+
+3. **`type` (string).**
+4. **`publicKeyJwk` (map[string,string], optional):**
+
+   A map representing a JSON Web Key that conforms to [RFC7517](https://tools.ietf.org/html/rfc7517). See definition of publicKeyJwk for additional constraints.
+
+5. **`publicKeyMultibase` (base58-encoded string, optional):**
+
+   A string that conforms to a [MULTIBASE](https://datatracker.ietf.org/doc/html/draft-multiformats-multibase-03) encoded public key.
+
+**Example:**
+```
+    {
+      "id": "N22KY2Dyvmuu2PyyqSFKue#key-0",
+      "type": "JsonWebKey2020",
+      "controller": "N22KY2Dyvmuu2PyyqSFKue",
+      "publicKeyJwk": {
+        "kty": "OKP", // external (property name)
+        "crv": "Ed25519", // external (property name)
+        "x": "VCpo2LMLhn6iWku8MKvSLg2ZAoC-nlOyPVQaO3FxVeQ" // external (property name)
+      }
 ```
 
-### ATTRIB
 
-Attributes to an existing DID record.
+#### Service
 
-* **`did` \(base58-encoded string\):**
+1. **`id` (string):**
 
-  Target DID as base58-encoded string for 16 or 32 byte DID value.
+   The value of the id property MUST be a URI conforming to [RFC3986](https://www.rfc-editor.org/rfc/rfc3986). A conforming producer MUST NOT produce multiple service entries with the same id. A conforming consumer MUST produce an error if it detects multiple service entries with the same id.
+   It has a follow format: `<DIDDoc-id>#<srvice-alias>`
 
-* **`raw` \(json; mutually exclusive with `hash` and `enc`\):**
+2. **`type` (strings):**
 
-  Raw data is represented as JSON, where the key is attribute name and value is attribute value.
+   the service type and its associated properties SHOULD be registered in the DID Specification Registries [DID-SPEC-REGISTRIES](https://www.w3.org/TR/did-spec-registries/).
 
-ATTRIB _transaction format:_:
+2. **`serviceEndpoint` (strings).**
 
-```text
-{
-              "did": "N22KY2Dyvmuu2PyyqSFKue",
-              "raw": "{"name": "Alice"}",
-}
-```
+   A string that conforms to the rules of [RFC3986](https://www.rfc-editor.org/rfc/rfc3986) for URIs, a map, or a set composed of a one or more strings that conform to the rules of [RFC3986](https://www.rfc-editor.org/rfc/rfc3986) for URIs and/or maps.
+
+**Example:**
 
-**Note:** ATTRIB **can** be updated
+    ```
+        "service": [{
+        "id":"N22KY2Dyvmuu2PyyqSFKue#linked-domain",
+        "type": "LinkedDomains",
+        "serviceEndpoint": "https://bar.example.com"
+        }]
+    ````
+
+#### **Update DID**
+
+If there is no DID transaction with the specified DID \(`dest`\), it is considered as a creation request for a new DID.
```

Comment:
`dest` looks like an outdated field.

Reply:
Yes I would agree. This is from the old request format.

Reply:
Fixed